pose-detection/src/calculators/keypoint_tracker.ts

```diff
@@ -32,10 +32,10 @@ export class KeypointTracker extends Tracker {
 
   constructor(config: TrackerConfig) {
     super(config);
-    //TODO(ronnyvotel): validate.
-    this.keypointThreshold = config.trackerParams.keypointConfidenceThreshold;
-    this.keypointFalloff = config.trackerParams.keypointFalloff;
-    this.minNumKeyoints = config.trackerParams.minNumberOfKeypoints;
+    this.keypointThreshold =
```

Comment:
I noticed you removed the config validation TODO. Do you want to do the validation somewhere else?